server/datastore/mysql/migrations/tables/20251006163522_InHouseAppsSupport.go

```diff
@@ -0,0 +1,58 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20251006163522, Down_20251006163522)
+}
+
+func Up_20251006163522(tx *sql.Tx) error {
+	createTableStmt := `
+CREATE TABLE in_house_apps (
+  id int unsigned NOT NULL AUTO_INCREMENT,
+  title_id int unsigned DEFAULT NULL,
+  team_id int unsigned DEFAULT NULL,
+  global_or_team_id int unsigned NOT NULL DEFAULT '0',
+  name varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
+  storage_id VARCHAR(64) COLLATE utf8mb4_unicode_ci NOT NULL,
+  created_at timestamp NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at timestamp NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  platform varchar(10) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
+  UNIQUE KEY (id),
+  PRIMARY KEY (id,platform),
```

Comment:
Nit, since `id` is unique and auto-increment, it should be `PRIMARY KEY`, and no need for `platform` to be part of it (if `id` is unique, `id, platform` is necessarily unique).

Reply:
or perhaps actually more helpful would be `global_or_team_id` first, then `name`, `platform` (so it's more like vpp apps and index can be used if we query by team only)